awx/settings/defaults.py

```diff
@@ -966,6 +966,7 @@ def IS_TESTING(argv=None):
 CHANNEL_LAYER_RECEIVE_MAX_RETRY = 10
 
 # Logging configuration.
+LOGGING_SOCK = '/var/run/tower/rsyslog/rsyslog.sock'
```

Comment:
I don't think this should be defined as an independent setting. If someone customizes the value in `/etc/tower/settings.py` (or wherever that is), then it will fail to take effect because the LOGGING dict still references the value set here. If you actually wanted to change it, you'd have to set `LOGGING['handlers']['external_logger']['address']` anyway. So might as well not have this intermediary variable so that people go straight to the gnarly syntax (which is the only correct option).

Reply:
I agree w/ @AlanCoding cc @rooftopcellist @shanemcd it would probably be better to just use `LOGGING['handlers']['external_logger']` everywhere and just reference it that way in the code